docs/tutorials/integrate-dotwallet.md

```diff
@@ -0,0 +1,142 @@
+---
+sidebar_position: 8
+---
+
+# Tutorial 8: Integrating DotWallet
+
+
+[DotWallet](https://www.dotwallet.com/en) is a lightweight wallet designed to help users easily and securely manage their digital assets. We will introduce how to integrate DotWallet.
+
+
+## Dotwallet Developer Platform
+
+DotWallet uses [oauth 2.0 protocol](https://oauth.net/2/) for login.
+
+1. Before using DotWallet, you need to register and create a dApp on DotWallet [Developer Platform](https://developers.dotwallet.com/en).
+
+![](../../static/img/dotwallet-create-dapp.png)
+
+2. After creating the dApp, you will receive an email containing `app_id` and `secret`.
+
+
+![](../../static/img/dotwallet-mail.png)
+
+3. Next, you need to set [redirect-uris](https://www.oauth.com/oauth2-servers/redirect-uris)
+
+For example, in the figure below, [https://classic.scrypt.io/tic-tac-toe](https://classic.scrypt.io/tic-tac-toe) is the address used for product deployment, [http:/ /localhost:3000/tic-tac-toe/](http://localhost:3000/tic-tac-toe/) is used for development and debugging.
```

Comment:
Explain 
- what is a callback domain here?
- why there are two callback domains we need to set?

Also explain what is https://classic.scrypt.io/tic-tac-toe and http:/ /localhost:3000/tic-tac-toe/.